float64 fuse_condition
